### PR TITLE
ReadMe changes for Chase Pay

### PR DESCRIPTION
**Pull request by sharathkatteri**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,6 +71,14 @@ Additionally, you can find details and examples of how our API is structured in
 The API Reference Guide provides examples of what information is needed for a particular request and how that information would be formatted. Using those examples, you can easily determine what methods would be necessary to include that information in a request using this SDK.
 
 
+## Create a Chase Pay Transaction
+Use this method to authorize and capture a payment using a tokenized credit card number issued by Chase Pay. Chase Pay is only available to merchants using the Paymentech processor, and you must pass in your request the token, expiration date, cryptogram receied from the token provider, tokenRequestorName, tokenRequestorId and tokenRequestorEci.
```
Comment (gnongsie) [inline README.md]: "received" spelling is incorrect.

Comment (gnongsie) [reply README.md]: This comment can be modified for easier readability. Use the following content:

> Use this method to authorize and capture a payment using a tokenized credit card number issued by Chase Pay. Chase Pay transactions are only available to merchants using the Paymentech processor.
> 
> The following information is required in the request:
> - The payment token, 
> - The expiration date, 
> - The cryptogram received from the token provider, 
> - The tokenRequestorName, 
> - The tokenRequestorId, and 
> - The tokenRequestorEci.

File: README.md [hunk 2]
```diff
@@ -71,6 +71,14 @@ Additionally, you can find details and examples of how our API is structured in
...
+
+When using the SDK to submit Chase Pay transactions, consider the following Points:
```
Comment (gnongsie) [inline README.md]: The word "points" should have a small case "p"

File: README.md [hunk 3]
```diff
@@ -71,6 +71,14 @@ Additionally, you can find details and examples of how our API is structured in
...
+- TokenRequesterName must be populate with  “CHASE_PAY”
```
Comment (gnongsie) [inline README.md]: The word should be "populated".

**Comment by gnongsie** [comment]
Ready to merge

**Comment by sharathkatteri** [comment]
Closing this

**Pull request closed without merging**